scripts/setup-ubuntu.sh

```diff
@@ -110,12 +109,20 @@ function cmake_install {
   ninja -C "${BINARY_DIR}" install
 }
 
+function install_fmt {
+  github_checkout fmtlib/fmt 7.1.3
+  cmake_install -DFMT_TEST=OFF
+}
+
 function install_folly {
   github_checkout facebook/folly "${FB_OS_VERSION}"
   cmake_install -DBUILD_TESTS=OFF
 }
 
 function install_velox_deps {
+  run_and_time install_fmt
+  sudo ln -s /usr/local/lib/libfmt.a /usr/lib/x86_64-linux-gnu/libfmt.a
```

Comment:
would like to hear anything better than doing this...

Reply:
Can we add `/usr/local` to `CMAKE_PREFIX_PATH`?

Reply:
Alternatively, if we defaulted to installing to `_build/prefix` this wouldn't be an issue at all.

Reply:
> Can we add /usr/local to CMAKE_PREFIX_PATH?

Make sense. Shall we just change the Makefile so we don't need this line? ^_^

Reply:
@funrollloops 
> Can we add /usr/local to CMAKE_PREFIX_PATH?

Thanks! Does that mean we need to do
```
CMAKE_PREFIX_PATH=/usr/local make debug
```
?

Is it possible to allow CMake to search in that path? (Sorry as a CMake noob :( )

Reply:
`CMAKE_PREFIX_PATH` is exactly how you tell CMake to look somewhere :)

Right now the Makefile does not allow you to pass through or modify the prefix path, but (1) that can be changed or (2) we can add `-DCMAKE_PREFIX_PATH=/usr/local` to the cmake flags in the Makefile. (2) is probably better, that way it will work out of the box -- which is the purpose of the setup script IMO.